internal/service/pushbasedlogexportapi/resource_schema.go

```diff
@@ -59,12 +59,12 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 
 type TFModel struct {
 	BucketName types.String `tfsdk:"bucket_name"`
-	CreateDate types.String `tfsdk:"create_date"`
-	GroupId    types.String `tfsdk:"group_id"`
+	CreateDate types.String `tfsdk:"create_date" autogeneration:"omitjson"`
+	GroupId    types.String `tfsdk:"group_id" autogeneration:"omitjson"`
 	IamRoleId  types.String `tfsdk:"iam_role_id"`
 	Links      types.List   `tfsdk:"links"`
```

Comment:
link does not have omitjson due to an inconsistency in the API Spec, will be following up on this